Models/Booking/Response/AvailabilityResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StockportGovUK.NetStandard.Models.Models.Booking.Response
+{
+    public class AvailabilityDayResponse
+    {
+        public DateTime Date { get; set; }
+        public List<AppointmentTime> AppointmentTimes { get; set; }
+        public bool HasAvailableAppointment => AppointmentTimes.Any();
+    }
```

Comment:
Our file name is AvailabilityResponse.cs but the main class is AvailabilityDayResponse, our file name and main class should share a name.